lib/presentation/feature/news/models/anime_news_item_model.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter_template_riverpod/data/model/api/response/get_top_anime_response.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'anime_news_item_model.freezed.dart';
+
+@freezed
+class AnimeNewsItemModel with _$AnimeNewsItemModel {
+  const factory AnimeNewsItemModel({
+    required int id,
+    required String titleEn,
+    required String titleJp,
+    required String imageUrl,
+  }) = _AnimeNewsItemModel;
+
+  factory AnimeNewsItemModel.fromGetTopAnimeResponse({
+    required GetTopAnimeResponseData responseData,
+  }) {
+    var firstImage = responseData.images?.jpg?.image_url;
```

Comment:
can we  declare final to avoid warning.
   final firstImage = responseData.images?.jpg?.image_url;